### ISSUE TITLE
【EESS】New submissions for Mon,  2 Oct 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#703]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
### Joint Task Offloading and Resource Allocation in Aerial-Terrestrial UAV  Networks with Edge and Fog Computing for Post-Disaster Rescue
 - **Authors:** Geng Sun, Long He, Zemin Sun, Qingqing Wu, Jiahui Li, Dusit Niyato, Victor C. M. Leung
 - **Subjects:** Signal Processing (eess.SP); Computer Science and Game Theory (cs.GT); Networking and Internet Architecture (cs.NI)
 - **Arxiv link:** https://arxiv.org/abs/2309.16709
 - **Pdf link:** https://arxiv.org/pdf/2309.16709
 - **Abstract**
 Unmanned aerial vehicles (UAVs) play an increasingly important role in assisting fast-response post-disaster rescue due to their fast deployment, flexible mobility, and low cost. However, UAVs face the challenges of limited battery capacity and computing resources, which could shorten the expected flight endurance of UAVs and increase the rescue response delay during performing mission-critical tasks. To address this challenge, we first present a three-layer post-disaster rescue computing architecture by leveraging the aerial-terrestrial edge capabilities of mobile edge computing (MEC) and vehicle fog computing (VFC), which consists of a vehicle fog layer, a UAV client layer, and a UAV edge layer. Moreover, we formulate a joint task offloading and resource allocation optimization problem (JTRAOP) with the aim of maximizing the time-average system utility. Since the formulated JTRAOP is proved to be NP-hard, we propose an MEC-VFC-aided task offloading and resource allocation (MVTORA) approach, which consists of a game theoretic algorithm for task offloading decision, a convex optimization-based algorithm for MEC resource allocation, and an evolutionary computation-based hybrid algorithm for VFC resource allocation. Simulation results validate that the proposed approach can achieve superior system performance compared to the other benchmark schemes, especially under heavy system workloads.
### Tree based Single LED Indoor Visible Light Positioning Technique
 - **Authors:** Srivathsan Chakaravarthi Narasimman, Arokiaswami Alphones
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2309.16974
 - **Pdf link:** https://arxiv.org/pdf/2309.16974
 - **Abstract**
 Visible light positioning(VLP) has gained prominence as a highly accurate indoor positioning technique. Few techniques consider the practical limitations of implementing VLP systems for indoor positioning. These limitations range from having a single LED in the field of view(FoV) of the image sensor to not having enough images for training deep learning techniques. Practical implementation of indoor positioning techniques needs to leverage the ubiquity of smartphones, which is the case with VLP using complementary metal oxide semiconductor(CMOS) sensors. Images for VLP can be gathered only after the lights in question have been installed making it a cumbersome process. These limitations are addressed in the proposed technique, which uses simulated data of a single LED to train machine learning models and test them on actual images captured from a similar experimental setup. Such testing produced mean three dimensional(3D) positioning error of 2.88 centimeters while training with real images achieves accuracy of less than one centimeter compared to 6.26 centimeters of the closest competitor.
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### T1/T2 relaxation temporal modelling from accelerated acquisitions using  a Latent Transformer
 - **Authors:** Fanwen Wang, Michael Tanzer, Mengyun Qiao, Wenjia Bai, Daniel Rueckert, Guang Yang, Sonia Nielles-Vallespin
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2309.16853
 - **Pdf link:** https://arxiv.org/pdf/2309.16853
 - **Abstract**
 Quantitative cardiac magnetic resonance T1 and T2 mapping enable myocardial tissue characterisation but the lengthy scan times restrict their widespread clinical application. We propose a deep learning method that incorporates a time dependency Latent Transformer module to model relationships between parameterised time frames for improved reconstruction from undersampled data. The module, implemented as a multi-resolution sequence-to-sequence transformer, is integrated into an encoder-decoder architecture to leverage the inherent temporal correlations in relaxation processes. The presented results for accelerated T1 and T2 mapping show the model recovers maps with higher fidelity by explicit incorporation of time dynamics. This work demonstrates the importance of temporal modelling for artifact-free reconstruction in quantitative MRI.
### General and Unified Model of the Power Flow Problem in Multiterminal  AC/DC Networks
 - **Authors:** Willem Lambrichts, Mario Paolone
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2309.16857
 - **Pdf link:** https://arxiv.org/pdf/2309.16857
 - **Abstract**
 This paper proposes a generic and unified model of the power flow (PF) problem for multiterminal hybrid AC/DC networks. The proposed model is an extension of the standard AC-PF. The DC network is treated as an AC one and, in addition to the Slack, PV and PQ nodes, four new node types are introduced to model the DC buses and the buses connecting the AC/DC interfacing converters (IC). The unified model is solved using the Newton-Raphson method. The extended PF equations can be used in the presence of multiple ICs operating under different control modes. Compared to other recent works, the proposed method allows multiple ICs to regulate the DC voltage simultaneously. This corresponds to more realistic operational conditions that ensure redundancy and allow for more flexible control of the hybrid grid. The proposed model can be used for networks under unbalanced conditions and allows for an intentionally negative sequence power injection. In addition to the operational advantages of this method, it is shown that the computational performance of the proposed method is one order of magnitude better than that of other methods presented in the existing recent literature while having the same accuracy.
### Towards High Resolution Weather Monitoring with Sound Data
 - **Authors:** Enis Berk Çoban, Megan Perra, Michael I. Mandel
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD)
 - **Arxiv link:** https://arxiv.org/abs/2309.16867
 - **Pdf link:** https://arxiv.org/pdf/2309.16867
 - **Abstract**
 Across various research domains, remotely-sensed weather products are valuable for answering many scientific questions; however, their temporal and spatial resolutions are often too coarse to answer many questions. For instance, in wildlife research, it's crucial to have fine-scaled, highly localized weather observations when studying animal movement and behavior. This paper harnesses acoustic data to identify variations in rain, wind and air temperature at different thresholds, with rain being the most successfully predicted. Training a model solely on acoustic data yields optimal results, but it demands labor-intensive sample labeling. Meanwhile, hourly satellite data from the MERRA-2 system, though sufficient for certain tasks, produced predictions that were notably less accurate in predict these acoustic labels. We find that acoustic classifiers can be trained from the MERRA-2 data that are more accurate than the raw MERRA-2 data itself. By using MERRA-2 to roughly identify rain in the acoustic data, we were able to produce a functional model without using human-validated labels. Since MERRA-2 has global coverage, our method offers a practical way to train rain models using acoustic datasets around the world.
### Reliability Quantification of Deep Reinforcement Learning-based Control
 - **Authors:** Hitoshi Yoshioka, Hirotada Hashimoto
 - **Subjects:** Systems and Control (eess.SY); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2309.16977
 - **Pdf link:** https://arxiv.org/pdf/2309.16977
 - **Abstract**
 Reliability quantification of deep reinforcement learning (DRL)-based control is a significant challenge for the practical application of artificial intelligence (AI) in safety-critical systems. This study proposes a method for quantifying the reliability of DRL-based control. First, an existing method, random noise distillation, was applied to the reliability evaluation to clarify the issues to be solved. Second, a novel method for reliability quantification was proposed to solve these issues. The reliability is quantified using two neural networks: reference and evaluator. They have the same structure with the same initial parameters. The outputs of the two networks were the same before training. During training, the evaluator network parameters were updated to maximize the difference between the reference and evaluator networks for trained data. Thus, the reliability of the DRL-based control for a state can be evaluated based on the difference in output between the two networks. The proposed method was applied to DQN-based control as an example of a simple task, and its effectiveness was demonstrated. Finally, the proposed method was applied to the problem of switching trained models depending on the state. Con-sequently, the performance of the DRL-based control was improved by switching the trained models according to their reliability.
### Benefits of mirror weight symmetry for 3D mesh segmentation in  biomedical applications
 - **Authors:** Vladislav Dordiuk, Maksim Dzhigil, Konstantin Ushenin
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2309.17076
 - **Pdf link:** https://arxiv.org/pdf/2309.17076
 - **Abstract**
 3D mesh segmentation is an important task with many biomedical applications. The human body has bilateral symmetry and some variations in organ positions. It allows us to expect a positive effect of rotation and inversion invariant layers in convolutional neural networks that perform biomedical segmentations. In this study, we show the impact of weight symmetry in neural networks that perform 3D mesh segmentation. We analyze the problem of 3D mesh segmentation for pathological vessel structures (aneurysms) and conventional anatomical structures (endocardium and epicardium of ventricles). Local geometrical features are encoded as sampling from the signed distance function, and the neural network performs prediction for each mesh node. We show that weight symmetry gains from 1 to 3% of additional accuracy and allows decreasing the number of trainable parameters up to 8 times without suffering the performance loss if neural networks have at least three convolutional layers. This also works for very small training sets.
### A Framework and a python-package for Real-time NMPC parameters settings
 - **Authors:** Mazen Alamir
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2309.17238
 - **Pdf link:** https://arxiv.org/pdf/2309.17238
 - **Abstract**
 This paper presents a framework that enables a systematic and rational choice of NMPC design components such as control updating period, down-sampling period for prediction, control parameterization, prediction horizon's length, the maximum number of iterations as well as penalties on the terminal cost and the soft constraints. The rationale that underlines the design choices is based on real-time implementability, convergence and constraints satisfaction for a given computational device and a specific optimization algorithm. Moreover, a freely available associated Python-based implementation is also described with a fully developed illustrative example implementing a nonlinear MPC controller for a Planar Vertical Take-Off and Landing (PVTOL) aircraft under control saturation and state constraints.
### Toward Universal Speech Enhancement for Diverse Input Conditions
 - **Authors:** Wangyou Zhang, Kohei Saijo, Zhong-Qiu Wang, Shinji Watanabe, Yanmin Qian
 - **Subjects:** Audio and Speech Processing (eess.AS); Sound (cs.SD); Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2309.17384
 - **Pdf link:** https://arxiv.org/pdf/2309.17384
 - **Abstract**
 The past decade has witnessed substantial growth of data-driven speech enhancement (SE) techniques thanks to deep learning. While existing approaches have shown impressive performance in some common datasets, most of them are designed only for a single condition (e.g., single-channel, multi-channel, or a fixed sampling frequency) or only consider a single task (e.g., denoising or dereverberation). Currently, there is no universal SE approach that can effectively handle diverse input conditions with a single model. In this paper, we make the first attempt to investigate this line of research. First, we devise a single SE model that is independent of microphone channels, signal lengths, and sampling frequencies. Second, we design a universal SE benchmark by combining existing public corpora with multiple conditions. Our experiments on a wide range of datasets show that the proposed single model can successfully handle diverse conditions with strong performance.
